### PR TITLE
new lesson about arrays

### PR DESCRIPTION
**Pull request by Johanna1988**

### CODE REVIEW

File: src/main/java/Arrays.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package main.java;
+
+public class Arrays {
+    public static void main(String[] args) {
+        int[] numbers = new int[5];
+        for (int i = 0; i < numbers.length; ++i) {
+            numbers[i] = i * 10;
+        }
```
Comment (Maks-Chornyi) [inline src/main/java/Arrays.java:8]: add empty line here for better visibility

File: src/main/java/Arrays.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        for (int i = 0; i < numbers.length; ++i) {
+            System.out.println(numbers[i]);
+        }
```
Comment (Maks-Chornyi) [inline src/main/java/Arrays.java:12]: same

File: src/main/java/Arrays.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        System.out.println();
+        int[] numbers1 = new int[]{1, 2, 3};
+
+        for (int i = 0; i < numbers1.length; ++i) {
+            System.out.println(numbers1[i]);
+
```
Comment (Maks-Chornyi) [inline src/main/java/Arrays.java]: line 17 is empty, you can delete it

File: src/main/java/Arrays.java (hunk 1 above)
Comment (Johanna1988) [reply src/main/java/Arrays.java:8]: done

File: src/main/java/Arrays.java (hunk 2 above)
Comment (Johanna1988) [reply src/main/java/Arrays.java:12]: done

File: src/main/java/Arrays.java (hunk 3 above)
Comment (Johanna1988) [reply src/main/java/Arrays.java]: deleted

**Review by Maks-Chornyi — APPROVED** [review]
Looks good. You can merge to main

**Pull request merged**